test/fixtures/convert/decorator/mobx/flow.js

```diff
@@ -0,0 +1,20 @@
+import { observable, computed, action } from "mobx";
+export class Store {
+  @observable
+  markersMapping: Map<number, MarkerStore> = new Map();
+
+  @computed
+  get filePath(): ?string {
+    const editor = this.editor;
+    if (!editor) return null;
+    const savedFilePath = editor.getPath();
+    return savedFilePath ? savedFilePath : `Unsaved Editor ${editor.id}`;
+  }
+
+  @action
+  newMarkerStore(editorId: number) {
+    const markerStore = new MarkerStore();
+    this.markersMapping.set(editorId, markerStore);
+    return markerStore;
+  }
+}
```

Comment:
I like that this test case covers a few different uses of decorators, but the implementation details add a bunch of noise that is distracting when comparing the input to the output.  Could you pare this test case down to the bare essentials?